enterprise_access/apps/enterprise_groups/management/commands/groups_reminder_emails.py

```diff
@@ -61,4 +61,13 @@ def handle(self, *args, **options):
                 pending_enterprise_customer_user["enterprise_customer_name"] = enterprise_customer_data["name"]
                 pending_enterprise_customer_user["enterprise_group_uuid"] = enterprise_group_uuid
                 pecu_email_properties.append(pending_enterprise_customer_user)
+
+            if not pecu_email_properties:
```

Comment:
[inform] No longer dispatches a Celery task if `pecu_email_properties` is empty. Otherwise, the task is spun up only to try to iterate over an empty list.